app/include/user_version.h

```diff
@@ -11,7 +12,7 @@
 #define NODE_VERSION_STR(x)	#x
 #define NODE_VERSION_XSTR(x)	NODE_VERSION_STR(x)
 
-#define NODE_VERSION		"NodeMCU " ESP_SDK_VERSION_STRING "." NODE_VERSION_XSTR(NODE_VERSION_INTERNAL)
+# define NODE_VERSION		"NodeMCU " ESP_SDK_VERSION_STRING "." NODE_VERSION_XSTR(NODE_VERSION_INTERNAL) " " NODE_VERSION_LONG
```

Comment:
Please a comment why (for the time being) that space shouldn't be removed. It's very fragile though and we should maybe find a less hacky solution (e.g. renaming `NODE_VERSION`).

Reply:
It will probably only be a short period of time, since you will want to have the vanity token in I guess.
But I will add it.

Reply:
Alternatively we could just remove the space again and have the docker build not benefit from this PR until it is adapted.
@marcelstoer What do you think? Whichever you prefer.

Reply:
Not sure...the Docker scripts still need to support NodeMCU that doesn't have this feature built in (`master` currently, `1.5.4.1`).

Reply:
You could adapt the script to act on the existence of `NODE_VERSION_LONG` at the end of the line. Or on `tools/update_buildinfo.sh`

Reply:
True, but we need to work on the Docker image anyhow (migrating to xenial) due to the `fatal: unknown date format format-local:%Y%m%d%H%M` issue.

Reply:
Hmm. The broken DTS will be in node.info anyhow, so It's just about the welcome message having the broken DTS or the docker format.
Ah and the vanity fair would be missing without docker build script change. So you might want to leave the docker welcome and I remove the space again.

Reply:
Yes, but the script execution will actually fail on Docker and thus fail the build.

Reply:
No it doesn't. I using it all the time.

Reply:
It prints an error massage but continues since the error is from a subshell and only breaks that.
So it only returns an empty DTS